Gopkg.toml

```diff
@@ -1,65 +1,20 @@
-[[constraint]]
-  name = "github.com/docker/go"
-  version = "1.5.1-1"
-
-[[constraint]]
-  name = "github.com/stretchr/testify"
-  version = "1.2.2"
-
-[[constraint]]
-  name = "gopkg.in/yaml.v2"
-  version = "2.2.1"
-
-[[constraint]]
-  # version imported by k8s.io/client-go @ kubernetes-1.11.2
-  name = "github.com/Azure/go-autorest"
-  revision = "1ff28809256a84bb6966640ff3d0371af82ccba4"
-
-[[constraint]]
-  name = "github.com/docker/cli"
-  revision = "f95ca8e1ba6c22c9abcdbf65e8dcc39c53958bba"
-
 [[override]]
-  name = "github.com/docker/docker"
-  revision = "f76d6a078d881f410c00e8d900dcdfc2e026c841"
+  name = "github.com/containerd/containerd"
+  version = "v1.3.0"
 
 [[override]]
   name = "github.com/docker/distribution"
-  revision = "83389a148052d74ac602f5f1d62f86ff2f3c4aa5"
-
-[[override]]
-  name = "k8s.io/apimachinery"
-  revision = "kubernetes-1.11.2"
-
-[[override]]
-  name = "k8s.io/api"
-  revision = "kubernetes-1.11.2"
+  #version = "v2.7.1"
```

Comment:
Why are we using that revision instead of v2.7.1? That commented out line is a bit misleading. Whenever we do pin to a specific revision it really helps to explain why in the file, so that people can understand when it's safe to move back to a tag, what issue they should be watching and waiting for it to be merged, etc.